gui/window.go

```diff
@@ -742,6 +742,31 @@ func (window *Window) launchMenuView() {
 	}
 }
 
+// writeToConfirmInstallDialog is a helper function to write to dialog for confirm installation window
+func writeToConfirmInstallDialog(buffer *gtk.TextBuffer, dryRunResults *storage.DryRunType) {
+	for _, media := range *dryRunResults.OtherMediaResults {
+		log.Debug("OtherMediaChange: %s", media)
+		buffer.Insert(buffer.GetEndIter(), media+"\n")
```

Comment:
Can we set these to the Warning color?
They do not stand out enough in the dialogue.

Reply:
@mdhorn: the only way to do this is to create two different TextView boxes one for things to stand out and others to not!  That way we can Add two different css classes on them

Reply:
Should we have two boxes? what do you feel?

Reply:
See replacement code below

Reply:
Done